starter-apps/vue/vue-template/src/views/HomeView.vue

```diff
@@ -0,0 +1,23 @@
+<script setup>
+import { useI18n } from 'vue-i18n'
+import Heading from '@/components/Heading.vue'
+import Text from '@/components/Text.vue'
+
+const { t } = useI18n()
+</script>
+<template>
+  <div>
+    <section>
+      <Heading class="mb-400" tag="h1">{{ t('homePage.heading') }}</Heading>
```

Comment:
The heading component should automatically set the `margin-bottom` to `400`, so there is no need to apply a utility class for this.